scripts/metadata/utils.py

```diff
@@ -0,0 +1,34 @@
+import pathlib
+from typing import Generator, List
+
+import tatm.data
+
+
+def metadata_files(
+    directory: pathlib.Path, include_tokenized=False, exclude_dirs: List[str] = None
+) -> Generator[tuple[str, pathlib.Path, tatm.data.TatmDataMetadata], None, None]:
```

Comment:
Could you add a docstring detailing what this function is supposed to find?

Reply:
Done